charts/ai-service/templates/artifacts-cache.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.artifactsCache.enabled -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.artifactsCache.name | default (include "common.fullname" .) }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    - {{ .Values.artifactsCache.accessMode }}
+  storageClassName: {{ .Values.artifactsCache.storageClassName }}
+  resources:
+    requests:
+      storage: {{ .Values.artifactsCache.storage }}
```

Comment:
you could default that to something sane of yours

Reply:
azurefile